README.md

```diff
@@ -56,15 +58,10 @@ sudo apt install libsqlite3-dev
 sudo apt install git
 ```
 Second, install the LDC compiler as per below:
-```text
-mkdir ldc && cd ldc
-wget http://ftp.us.debian.org/debian/pool/main/l/ldc/ldc_1.8.0-3_i386.deb
-wget http://ftp.us.debian.org/debian/pool/main/l/ldc/libphobos2-ldc-shared-dev_1.8.0-3_i386.deb
-wget http://ftp.us.debian.org/debian/pool/main/l/ldc/libphobos2-ldc-shared78_1.8.0-3_i386.deb
-wget http://ftp.us.debian.org/debian/pool/main/l/llvm-toolchain-5.0/libllvm5.0_5.0.1-2~bpo9+1_i386.deb
-wget http://ftp.us.debian.org/debian/pool/main/n/ncurses/libtinfo6_6.1+20180714-1_i386.deb
-sudo dpkg -i ./*.deb
-```
+1. Add `deb http://ftp.us.debian.org/debian sid main` to your `/etc/apt/sources.list`
```

Comment:
That is not really a good idea. It would upgrade **all** packages from say stable to unstable, which is definitely a no-go.

I am not sure what to recommend here. Probably best is to recommend building with dmd instead of ldc on older systems, backporting all the dependency to get ldc installed is I think not an option.